config_defaults_inc.php

```diff
@@ -3748,13 +3748,6 @@
 	'my_comments'   => '0'
 );
 
-/**
- * Toggle whether 'My View' boxes are shown in a fixed position (i.e. adjacent
- * boxes start at the same vertical position)
- * @global integer $g_my_view_boxes_fixed_position
- */
-$g_my_view_boxes_fixed_position = ON;
```

Comment:
Remove also from `$g_public_config_names`

Reply:
I was sure I had done this, must have forgotten to commit the change... Thanks